server/botany/settings.py

```diff
@@ -23,8 +23,20 @@
 BOTANY_NUM_ROUNDS = int(os.environ["BOTANY_NUM_ROUNDS"])
 BOTANY_OPCODE_LIMIT = int(os.environ["BOTANY_OPCODE_LIMIT"])
 
-# TODO read this from environment (and make optional)
-BOTANY_TOURNAMENT_CLOSE_AT = datetime(2018, 9, 18, 14, 30, tzinfo=bst)
+# Tournament time constraints in format: '%d/%m/%Y %H:%M'
+# eg. 04/11/2010 16:30.
+# If not given, +/- 1 day is a default.
+BOTANY_TOURNAMENT_START_AT = os.environ.get("BOTANY_TOURNAMENT_START_AT")
+if BOTANY_TOURNAMENT_START_AT is None:
+    BOTANY_TOURNAMENT_START_AT = datetime.now(timezone.utc) - timedelta(1)
+else:
+    BOTANY_TOURNAMENT_START_AT = datetime.strptime(BOTANY_TOURNAMENT_START_AT, '%d/%m/%Y %H:%M')
```

Comment:
The parsed datetime will not have a time zone (a naive datetime) which will lead to issues later when you compare this with values that do.

Also, can I suggest that [`datetime.fromisoformat()`](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat) is used to parse the string instead? ISO8601 is a more widely understood format for this and leaves open the possibility to leave out the time, or specify seconds, and include a timezone offset, etc. 

Then, if no time zone is specified then UTC could be attached.